src/unit_tests/test_syscheck_config.c

```diff
@@ -24,6 +24,13 @@ int __wrap__merror()
     return 0;
 }
 
+static int restart_syscheck(void **state)
+{
+    (void) state;
+    memset(&syscheck, 0, sizeof(syscheck_config));
```

Comment:
Restarting syscheck in this way causes several memory leaks.

Reply:
I fixed this.